portal-impl/src/com/liferay/portal/events/ServicePreAction.java

```diff
@@ -1982,7 +1982,9 @@ private void _updateUserLayouts(User user) throws Exception {
 		if (PrefsPropsUtil.getBoolean(
 				user.getCompanyId(),
 				PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_ENABLED) &&
-			PropsValues.LAYOUT_USER_PRIVATE_LAYOUTS_AUTO_CREATE) {
+			PrefsPropsUtil.getBoolean(
+				user.getCompanyId(),
+				PropsKeys.LAYOUT_USER_PRIVATE_LAYOUTS_AUTO_CREATE)) {
```

Comment:
@torresvictor100, incorrect commit message. You've used `Reclace` and `Replace`.